drivers/led_strip/apa102.c

```diff
@@ -74,6 +76,32 @@ static int apa102_update_rgb(const struct device *dev, struct led_rgb *pixels,
 	return apa102_update(dev, pixels, sizeof(struct led_rgb) * count);
 }
 
+/*
+ * Update the strip with the given channel values.
+ * Channels are expected to be ordered as follows for each LED:
+ *   - Dimming (5 bits)
+ *   - Blue (1 byte)
+ *   - Green (1 byte)
+ *   - Red (1 byte)
+ */
```

Comment:
@simonguinot here is the header, is this what you expected?

Reply:
I was thinking more of a header file (e.g. `apa102.h`) with channel definitions than a comment.